smart_meter_texas/__init__.py

```diff
@@ -35,9 +36,9 @@
     SmartMeterTexasRateLimitError,
 )
 
-__author__ = "Graham Wetzler"
-__email__ = "graham@wetzler.dev"
-__version__ = "0.4.7"
+__author__ = "Bobbie Turner"
```

Comment:
Please don't overwrite these. I will bump the version using a cli tool.

Reply:
Updated, sorry.